tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -531,14 +528,15 @@ private function createContext(): Context
             // We assume the internal client URI has multiple mongos hosts
             $multiMongosUri = $this->internalClientUri;
 
-            /* TODO: If an SRV URI is provided, we can consider connecting and
-             * checking the topology for multiple mongoses and then selecting a
-             * single mongos to reconstruct a single mongos URI; however, that
-             * may omit necessary URI options provided by TXT records. */
-            assertStringStartsWith('mongodb://', $multiMongosUri);
-            assertStringContainsString(',', parse_url($multiMongosUri, PHP_URL_HOST));
-
-            $singleMongosUri = self::removeMultipleHosts($multiMongosUri);
+            if (strpos($multiMongosUri, 'mongodb+srv://') === 0) {
+                /* TODO: If an SRV URI is provided, we can consider connecting and
+                 * checking the topology for multiple mongoses and then selecting a
+                 * single mongos to reconstruct a single mongos URI; however, that
+                 * may omit necessary URI options provided by TXT records. */
+                $singleMongosUri = $multiMongosUri;
```

Comment:
@alcaeus: How does this address [PHPLIB-1037](https://jira.mongodb.org/browse/PHPLIB-1037)? I'd expect this could introduce problems for tests that specify `useMultipleMongoses: false`, since they'd now run against a multi-mongos URI. Is that not the case?

Reply:
On atlas, we're not running any tests with useMultipleMongoses of any value. That said, the assertion makes a number of tests fail as it's testing a sharded cluster with a mongodb+srv connection string. Removing the assertions fixes those tests without introducing any other test failures.